### DISCUSSION TITLE
Unable to copy repository

### DISCUSSION THREAD

**Discussion by mbaca2020** [Guilouz/Creality-Helper-Script-Wiki#612] [Issues]
Is there any workaround to install with the error in the screenshot I took.
![Screenshot 2024-06-14 142941](https://github.com/Guilouz/Creality-Helper-Script-Wiki/assets/172847931/849aad4f-6ef1-4fb3-89c2-00b1d2048e82)

**Comment by pauloeduardogodoy** [comment]
If I'm not mistaken this is a timestamp error. Just follow the wiki instructions and fire up some commands and you are going to be good to go.
